client/src/components/home/HomeComponent.tsx

```diff
@@ -61,59 +38,74 @@ export const HomeComponent = (props: HomeComponentProps): JSX.Element => {
-                iconProps={videoIconStyles(props.theme)}
-                aria-roledescription="split button"
-                menuProps={menuProps(props)}
-              >
-                <Text styles={buttonTextStyles}>Start a call</Text>
-              </DefaultButton>
-              <DefaultButton
-                styles={buttonStyles}
-                iconProps={linkIconStyles(props.theme)}
-                onClick={() => window.location.assign('/visit')}
-              >
-                <Text styles={buttonTextStyles}>Join from link</Text>
-              </DefaultButton>
-            </Stack>
+            <HomeButtons theme={theme} setError={props.onDisplayError} />
             <FrequentlyAskedQuestions />
-            <Text styles={font16pxStyle}>Learn more about Azure Communication Services</Text>
-            <LearnMoreItem
-              headerText={'Azure Communication Services virtual appointments'}
-              headerLink={'https://learn.microsoft.com/azure/communication-services/tutorials/virtual-visits'}
-              description={
-                'This tutorial describes concepts for virtual appointment applications. After completing this tutorial and the associated Sample Builder, you will understand common use cases that a virtual appointment delivers...'
-              }
-            />
-            <LearnMoreItem
-              headerText={'Azure Communication Services Rooms (Preview)'}
-              headerLink={'https://learn.microsoft.com/en-us/azure/communication-services/concepts/rooms/room-concept'}
-              description={
-                'Azure Communication Services provides a concept of a room for developers who are building structured conversations such as virtual appointments or virtual events. Rooms currently allow voice and video calling...'
-              }
-            />
-            <LearnMoreItem
-              headerText={'Get Started with UI Library'}
-              headerLink={
-                'https://learn.microsoft.com/azure/communication-services/quickstarts/ui-library/get-started-composites?tabs=kotlin&pivots=platform-web'
-              }
-              description={
-                'Get Started with Azure Communication Services UI Library to quickly integrate communication experiences into your applications. In this quickstart, learn how to integrate UI Library composites into an application and set up...'
-              }
-            />
+            <LearnMore />
           </Stack>
         </Stack>
       </Stack>
     </Stack>
   );
 };
+
+const HomeButtons = ({ theme, setError }): JSX.Element => {
+  const buttonStyles = getButtonStyles(theme);
+
+  const callCreateRoom = async (): Promise<void> => {
+    try {
+      const redirectUrl = await createRoomAndRedirectUrl();
+      window.location.assign(redirectUrl);
+    } catch (error) {
+      setError(error);
+    }
+  };
+
+  const HomeButton = ({ iconName, text, onClick }): JSX.Element => {
```

Comment:
should we move this out or is it okay to leave it inside of the HomeButtons component

Reply:
I'll move it out